### DISCUSSION TITLE
add auto localization tool for docs

### DISCUSSION THREAD

**Discussion by uvicorn** [fastapi/fastapi#6971] [Questions] [answered]
I find a [tool to localize docs](https://github.com/apps/gitlocalize-app) .How about trying to add it to this repo?

**Comment by tiangolo** [comment, marked as answer]
Thanks, there's now a system to handle translations by users: https://fastapi.tiangolo.com/contributing/#translations

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
